tests/meta/upload/conftest.py

```diff
@@ -128,6 +129,35 @@ def convert_panels(customer_id, panels):
         return ""
 
 
+class MockCoverage:
```

Comment:
```suggestion
class MockCoverage(ChanjoAPI):
```
Would this be useful?